README.rst

```diff
@@ -63,34 +63,45 @@ include a set of functional tests written to use a real BIG-IP® device
 for testing. Information on how to run our set of tests is included
 below.
 
-Unit Tests
-==========
-
-We use pytest for our unit tests.
+Style Checks
+============
 
-1. If you haven't already, install the required test packages and the
-   requirements.txt in your virtual environment.
+We use the hacking module for our style checks.
 
 ::
+    $ pip install hacking
+    $ flake8 ./f5_openstack_agent
```

Comment:
Is this still appropriate, meant to replace with tox?  You created a new requirements.style.txt, so could do tox -e style.  But I don't see that env defined.